### DISCUSSION TITLE
Official Discord Channel

### DISCUSSION THREAD

**Discussion by lrljoe** [rappasoft/laravel-livewire-tables#1252] [General]
If you fancy dropping in to either discuss an issue you're having, or to show off your implementation, then feel free to drop in to the official channel.

[https://discord.com/invite/pQgzQuRZuS](https://discord.com/invite/pQgzQuRZuS)

**Comment by Omranic** [comment]
Couldn't join, it's saying "Whoops, Unable to accept invite."

<img width="496" alt="Screenshot 2023-09-08 at 12 08 58 PM" src="https://github.com/rappasoft/laravel-livewire-tables/assets/406705/40e3a54d-e6d9-481e-ae08-9627cbfad7c2">

**Reply by juanborras** [reply]
Same here

**Reply by lrljoe** [reply]
https://discord.com/invite/pQgzQuRZuS

**Comment by theticraft** [comment]
<img width="485" height="367" alt="image" src="https://github.com/user-attachments/assets/2e781cfe-dc55-4196-9179-010fc019e67b" />

It is invalid 🤔 

I would more or less like to contribute to the German translation. I noticed that the sentence structure isn't quite right.
